src/topology/sheaves/sheaf.lean

```diff
@@ -136,12 +136,113 @@ def fork.iso_of_iso (α : F ≅ G) :
   fork F U ≅ (cones.postcompose (diagram.iso_of_iso U α).inv).obj (fork G U) :=
 begin
   fapply fork.ext,
-  apply α.app,
-  ext,
-  dunfold fork.ι, -- Ugh, it is unpleasant that we need this.
-  simp [res, diagram.iso_of_iso],
+  { apply α.app, },
+  { ext,
+    dunfold fork.ι, -- Ugh, it is unpleasant that we need this.
```

Comment:
Can you not throw it into the `simp`?

Reply:
The problem is that `fork.\io` is an abbreviation, and `simp` won't unfold those. It's a bit unpleasant really. Usually `simp` just sees right through the abbreviations but this time is doesn't. :-(

Reply:
Interesting!